Method: exceptiontrycatchfinallywhen!Program.<<Main>$>g__Throw|0_0()

Trace 0
	[start] exceptiontrycatchfinallywhen!Program.<<Main>$>g__Throw|0_0()
		[start] exceptiontrycatchfinallywhen!Program.<Main>$(class System.String[])
		[ end ] exceptiontrycatchfinallywhen!Program.<Main>$(class System.String[])
	[ end ] exceptiontrycatchfinallywhen!Program.<<Main>$>g__Throw|0_0()

Method: exceptiontrycatchfinallywhen!Program.<Main>$(class System.String[])

Trace 0
	[start] exceptiontrycatchfinallywhen!Program.<Main>$(class System.String[])
	[ end ] exceptiontrycatchfinallywhen!Program.<Main>$(class System.String[])
Trace 1
	[start] exceptiontrycatchfinallywhen!Program.<Main>$(class System.String[])
		[start] exceptiontrycatchfinallywhen!Program.<<Main>$>g__Throw|0_0()
			[start] exceptiontrycatchfinallywhen!Program.<Main>$(class System.String[])
			[ end ] exceptiontrycatchfinallywhen!Program.<Main>$(class System.String[])
		[ end ] exceptiontrycatchfinallywhen!Program.<<Main>$>g__Throw|0_0()
	[ end ] exceptiontrycatchfinallywhen!Program.<Main>$(class System.String[])

